README.md

```diff
@@ -21,47 +20,67 @@ Version control is required for all projects. At DH we work with git and GitHub.
        'Add new About Us page' or 'Refactor tests for the order model'
     3. It should correctly complete the sentence: "If accepted, this commit will <your commit message goes here>."
 
+## Python package
+To prevent having to add the `src/` folder to the system path, it's often easier to structure your code as a Python package.
+This project template follows the `src/` package structure, where each folder in the `src` folder is a Python package.
+To locally install your package use:
+```{bash}
+pip install -e .
+```
+This creates an editable install, which means that after you make changes to your local package, you don't have to reinstall the package to use it.
+The standard template also contains optional dependencies for testing and developing. To also install the optional dependencies use: `pip install -e ".[dev]"` or `pip install -e ".[test]"`.
+
+If your package is called `my_package` you can import it anywhere by using:
+```{python}
+import my_package
+from my_package import my_function
+```
+
+## Specifying dependencies
+You can specify dependencies in the `pyproject.toml` either as normal dependecies, or optional dependencies (like dev and test dependencies).
+That way these packages will always be installed along with your package.
+
+It is recommended to also use a requirements.txt file to pin specific package versions, which makes contributing and deployment easier. But don't forgot to update these versions regularly.
 
 ## Virtual environment
 ### Using conda
-```
+```{bash}
 conda env create -f environment.yml
 conda activate
 ```
 
 ### Using venv
-```
+```{bash}
 python3 -m venv venv
 . venv/bin/activate
 pip install -r requirements.txt
 ```
 
-To add your local packages to you pythonpath run 
-`export PYTHONPATH="${PYTHONPATH}:."`
-
 ## Documentation and styleguide
 Styleguids can be checked with linters, for instance, `flake8`.
 By default, we adhere to the PEP-8 conventions
 The line-length is automatically formatted by `black`
-and has a maximum of 88. Line lengths of upto 119 characters are permissible.
-The `setup.cfg` files contains guidelines we choose to ignore.
+and has a maximum of 88.
+The import are sorted by isort (in vscode: rightclick `sort imports` or auto organise imports on save)
+The `setup.cfg` files contains the setup for flake8, including ignored folder (like tests/ and notebooks/).
```

Comment:
@rubenpeters91 dit kan dus niet in de pyproject.toml? suf...

Reply:
Jep, Flask is een van de weinige die nog niet in pyproject.toml kan...